src/mmrelay/migrate.py

```
@@ -0,0 +1,2511 @@
+                                continue
+                    except ValueError:
+                        continue
+        except (OSError, IOError, subprocess.TimeoutExpired, FileNotFoundError):
+            pass
+    else:
+        # On Windows, try using tasklist
+        try:
+            tasklist_path = shutil.which("tasklist")
+            if not tasklist_path:
+                return False
+            result = subprocess.run(  # nosec B603,B607
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)